src/ClientRuntime/Microsoft.Rest.ClientRuntime.E2E.Tests/project.json

```diff
@@ -0,0 +1,56 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "Microsoft Rest ClientRuntime End to End Tests",
+  "authors": [ "Microsoft Corporation" ],
+
+  "packOptions": {
+    "summary": "TestFramework.Tests Class Library",
+    "tags": [ "" ],
+    "projectUrl": "",
+    "licenseUrl": ""
+  },
+
+  "buildOptions": {
+    "delaySign": true,
+    "publicSign": false,
+    "keyFile": "../../../tools/MSSharedLibKey.snk",
+    "compile": "../../../tools/DisableTestRunParallel.cs"
+  },
+
+  "testRunner": "xunit",
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [ "dnxcore50", "portable-net45+win8" ],
+      "dependencies": {
+      }
+    }
+  },
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0"
+    },
+    "Microsoft.Azure.Test.HttpRecorder": {
+      "target": "project",
+      "type": "build"
+    },
+    "Microsoft.Rest.ClientRuntime.Azure.TestFramework": {
+      "target": "project",
+      "type": "build"
+    },
```

Comment:
please decouple ClientRutnimes tests and TestFramework. Do a nuget package reference instead.